lib/config/definitions.js

```diff
@@ -8,8 +8,8 @@ function template(name, subdir = 'default/') {
     path.resolve(__dirname, '../config/templates/', subdir, shortName),
     'utf8'
   );
-  // Strip off any trailing line break
-  return hbsContents.replace(/\n$/, '');
+  // Strip off any trailing line break (handle for Macs, Linux, Windows)
+  return hbsContents.replace(/\r?\n|\r/g, '');
```

Comment:
Can you break this into a separate PR? I like to keep unrelated changes to a minimum

Reply:
Yes, I'll look at it